routers/repo/http.go

```diff
@@ -176,6 +178,13 @@ func HTTP(ctx *context.Context) {
 					ctx.Handle(http.StatusInternalServerError, "GetUserByID", err)
 					return
 				}
+			} else {
+				_, err = models.GetTwoFactorByUID(authUser.ID)
+
+				if err == nil {
```

Comment:
You need to also check that `if !models.IsErrTwoFactorNotEnrolled(err)` and do `ctx.Handle(http.StatusInternalServerError, "IsErrTwoFactorNotEnrolled", err) + return` otherwise if there is some kind of unexpected error you will allow user in